### PR TITLE
updating fisher_swbd nnet3/chain scripts: removing non-chain training…

### PR DESCRIPTION
**Pull request by xiaohui-zhang**
… scripts, updating ivector training scripts,.etc. @GaofengCheng please re-run and check in your latest chain model training scripts on top of this. Thanks.

### CODE REVIEW

File: egs/fisher_swbd/s5/local/fisher_swbd_prepare_dict.sh [hunk 1]
```diff
@@ -112,8 +112,12 @@ srcdict=$srcdir/swb_ms98_transcriptions/sw-ms98-dict.text
 # assume swbd_p1_data_prep.sh was done already.
 [ ! -f "$srcdict" ] && echo "No such file $srcdict" && exit 1;
 
+rm -f $dir/lexicon0.txt
```
Comment (danpovey) [inline egs/fisher_swbd/s5/local/fisher_swbd_prepare_dict.sh]: I don't like using `-f` to make rm quiet.  do rm with `2>/dev/null`.

File: egs/fisher_swbd/s5/local/fisher_swbd_prepare_dict.sh [hunk 2]
```diff
@@ -112,8 +112,12 @@ srcdict=$srcdir/swb_ms98_transcriptions/sw-ms98-dict.text
...
 cp $srcdict $dir/lexicon0.txt || exit 1;
-patch <local/dict.patch $dir/lexicon0.txt || exit 1;
+chmod +x $srcdict $dir/lexicon0.txt
```
Comment (danpovey) [inline egs/fisher_swbd/s5/local/fisher_swbd_prepare_dict.sh]: This doesn't make sense to me.  why make them executable?

Comment (xiaohui-zhang) [reply egs/fisher_swbd/s5/local/fisher_swbd_prepare_dict.sh]: sorry I meant +w. will fix

**Comment by GaofengCheng** [comment]
@xiaohui-zhang will do

**Pull request merged**